src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Red Hat Ansible Automation Controller integration
+tags:
+   - Ansible controller  integration
```

Comment:
Add tag for Red Hat Ansible Automation Controller Integration

Reply:
done!